.tool-versions

```diff
@@ -1,2 +1,2 @@
 nodejs 16.2.0
-golang 1.18.1
```

Comment:
Let's leave this as 1.18.1. My thinking is:
1. Our general policy is to support the last two minor versions of Go (and pretending that X.Y.0 doesn't exist because it's frequently buggy) which means we should be able to build with the latest version.
2. I don't want one project being slow to cause others to be slow

Ideally I think we'd support building with two Go versions (in this case 1.17 and 1.18) and allow some level of configuration around what we build with (e.g. if we really wanted to do a patch release for a binary only built for 1.17 we'd currently be sunk).

cc @hsanjuan for thoughts here

Reply:
agree about leaving golang 1.18.1 here.